CodeEdit/Features/AppPreferences/Sections/TextEditingPreferences/TextEditingPreferencesView.swift

```diff
@@ -75,6 +75,9 @@ struct TextEditingPreferencesView: View {
             PreferencesSection("Line Wrapping") {
                 wrapLinesToEditorWidth
             }
+            PreferencesSection("Default Text Decoding") {
```

Comment:
It should be called `Default Text Encoding`. See Xcode or any other editor as a reference.

<img width="409" alt="Screenshot 2023-03-25 at 10 26 18" src="https://user-images.githubusercontent.com/9460130/227708873-db07923f-f89b-4a6c-b2d8-055357e85db6.png">